lib/authentic/validator.rb

```diff
@@ -21,20 +21,28 @@ def self.valid?(token)
   #
   # Returns nothing.
   def self.ensure_valid(token)
-    Validator.new.ensure_valid(token)
+    Validator.instance.ensure_valid(token)
   end
 
   # Public: validates JWTs against JWKs.
   class Validator
+    include Singleton
```

Comment:
Another way to do this without using Singleton would be to store `manager` and `iss_whitelist` as class variables that get set with a class method `configure`.  You would then need to throw an error in the initialize method if a manager and iss_whitelist were not set.

Usage would look like:
```
Authentic::Validator.configure(opts)
Authentic::Validator.new.valid?(token)
```